### ISSUE TITLE
Bump nodemon version to fix vulnerability

### ISSUE THREAD

**Issue by bpehling** [bee-queue/arena#228]
```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ dot-prop                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=5.1.1                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ bull-arena                                                   │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ bull-arena > nodemon > update-notifier > configstore >       │
│               │ dot-prop                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/1213                            │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

Latest nodemon `v2.0.4` contains the patched version of dot-prop

**Comment by bpehling** [comment]
@skeggse Any idea when a new version will be released to include the fix? We are blocked from using this and have ripped it out in the meantime

**Comment by skeggse** [comment]
Released in v2.8.2.
